src/actions/baseAction.ts

```diff
@@ -52,7 +52,9 @@ module Plywood {
     lookup?: string;
     attributes?: string[];
     simple?: boolean;
-    transformType?: CaseType;
+    transformCaseType?: CaseType;
+    transformFnName?: string;
+    transformType?: PlyTypeSingleValue;
```

Comment:
For these attribute names I was thinking of having them more reused, trying to get actions to share them more. Partially to keep the size of this interface down, partially to have less stuff to document. Also to keep these names shorter if possible (this JSON is sent back and forth to the server)

I was thinking:
- use `custom` instead of `transformFnName` as in "the custom thing that this action needs"
- use `outputType` instead of `transformType` for the output type of the CastAction
- keep `transformType` for TransformCase action the same, do not change it
- thinking about it now, ideally we would rename `castType` to `outputType` and outputType can be the thing that every future action uses if it needs to define an output type.

If you rename a property of an existing action, make sure to add back compat code and tests.